lib/run-contentful-export.js

```diff
@@ -22,6 +23,15 @@ export default function runContentfulExport (usageParams) {
     spaceId: clients.source.spaceId
   })
     .then((response) => {
+      if (opts.downloadAssets) {
+        log.info('Downloading ' + response.assets.length + ' assets')
+        response.assets.forEach((asset) => {
```

Comment:
Based on my previous comment about Promises, if the function returns a promise her you can use `Array.map` ->  `arrayOfDownloadPromises = response.assets.map((asset) => { ... ` and then you can call `Promise.all(arrayOfDownloadPromises)`